Space-X Defense Program/Space-X Defense Program/GameScene.swift

```diff
@@ -11,63 +11,87 @@ import GameplayKit
 
 class GameScene: SKScene {
     
-    //hello world
+    private let background = SKSpriteNode(imageNamed: "background")
+    private let player = Player()
+    let projectiles = Projectile()
     
     private var label : SKLabelNode?
     private var spinnyNode : SKShapeNode?
     
+    var gameTimer: Timer!                     //timer for repeating functions
+    
     override func didMove(to view: SKView) {
         
+        backgroundColor = SKColor.black       //defaulted colored to black for the background
+        background.zPosition = -1             //depth of background, -1 makes it go behind other 2D elements
+        background.position = CGPoint(x: size.width / 2, y: size.height / 2 )          //centering the background images
+        background.setScale(1.43)             //Maximizing the pixel resolution for the 1080p
```

Comment:
All the hardcoded values should be named and put in a struct for reusability and easy lookup. etc. This will also make it easier to change the values without having to search through the implementation when testing.
```
extension GameScene {
     static let bgScale = 1.43
     static let playerStartPosition = CGPoint(x: 220, y: 400) 
     static let playerScale = 0.6
}
```